blocks/sidekick-generator/sidekick-generator.js

```diff
@@ -37,15 +37,28 @@ function run(evt) {
     return;
   }
 
+  let finalGitUrl = giturl;
+  const giturlAsUrl = new URL(giturl);
+  if (giturlAsUrl.hostname.endsWith('hlx.page') || giturlAsUrl.hostname.endsWith('hlx.live')) {
```

Comment:
Smart! I think that could also be useful in the "add manually" form on the extension's option page.